conans/client/generators/cmake_common.py

```diff
@@ -169,16 +169,22 @@ def generate_targets_section(dependencies):
 
 _cmake_common_macros = """
 
+function(conan_message MESSAGE_OUTPUT)
+    if(NOT CONAN_CMAKE_SILENT_OUTPUT)
```

Comment:
FATAL_ERROR shouldn't be silenced. Check here arguments or just leave the callers as they were.